drivers/spi/spi_dw.c

```diff
@@ -521,9 +521,6 @@ int spi_dw_init(const struct device *dev)
 	const struct spi_dw_config *info = dev->config;
 	struct spi_dw_data *spi = dev->data;
 
-	clock_config(dev);
```

Comment:
Can the #include <drivers/clock_control.h> be removed, without the clock control?

Reply:
removed the include.